iOS_Labs/ViewModels/SignUp/SignUpViewModel.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  SignUpModel.swift
+//  iOS_Labs
+//
+//  Created by Daniil Oleshchuk on 30.10.2021.
+//
+
+import Foundation
+
+
+class SignUpViewModel: ObservableObject {
```

Comment:
This should be already merged in scope of lab3. So PR will be easier to review with content of only lab4.